crypto/kzg4844/kzg4844.go

```diff
@@ -61,6 +63,8 @@ func UseCKZG(use bool) error {
 	} else {
 		gokzgIniter.Do(gokzgInit)
 	}
+	// Compute the precompile return value using go-kzg
```

Comment:
Please don't default to one or the other, if there's a bug in one of the libs, we made them swappable for a reason.